src/prelexer.cpp

```diff
@@ -1260,6 +1260,9 @@ namespace Sass {
     const char* kwd_lte(const char* src) {
       return exactly<lte>(src);
     }
+    const char* kwd_using(const char* src) {
+      return keyword < using_kwd >(src);
```

Comment:
Can we not put spaces around `<template>`? This isn't ever necessary in C++11.

Reply:
I agree it's not necessary but template space (at least between < >) is
consistent throughout much of the codebase. Can't speak for anyone else
but I've personally found it makes the code more easily scannable.

Curious to know about your objection to the style?

Reply:
Updated to match the adjacent style.